src/python/pants/backend/jvm/subsystems/jvm.py

```diff
@@ -54,7 +54,9 @@ def get_jvm_options(self):
 
     Thus named because get_options() already exists (and returns this object's Pants options).
     """
-    ret = []
+    # Always bump MaxFDLimit - this means that Java will listen to the environmental FD limit,
+    # rather than imposing its own. This should just be what the JVM does by default. Boo.
```

Comment:
It's hard to come back from making this the default globally like this... are we certain that we want this to be un-removable? Although it would have less "automatic" impact, adding this as the default for the `--options` would also allow it to be disabled.

Reply:
... 9 months later, done!